.golangci.yaml

```diff
@@ -44,6 +45,194 @@ issues:
+        alias: gardencoreclientset
+      - pkg: github.com/gardener/gardener/pkg/client/core/clientset/versioned
+        alias: gardencoreversionedclientset
+      - pkg: github.com/gardener/gardener/pkg/client/core/informers/externalversions
+        alias: gardencoreexternalinformers
+      - pkg: github.com/gardener/gardener/pkg/client/core/informers/internalversion
+        alias: gardencoreinformers
+      - pkg: github.com/gardener/gardener/pkg/client/core/listers/core/internalversion
+        alias: gardencorelisters
+      - pkg: github.com/gardener/gardener/pkg/client/core/listers/core/v1alpha1
+        alias: gardencorev1alpha1listers
+      - pkg: github.com/gardener/gardener/pkg/client/kubernetes
+        alias: kubernetesclient
+      - pkg: github.com/gardener/gardener/pkg/client/kubernetes/(\w+)
+        alias: kubernetes${1}
+      - pkg: github.com/gardener/gardener/pkg/client/kubernetes/clientmap/builder
+        alias: clientmapbuilder
+      - pkg: github.com/gardener/gardener/pkg/client/kubernetes/clientmap/fake
+        alias: fakeclientmap
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/clientset/versioned
+        alias: seedmanagementclientset
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/informers/externalversions
+        alias: seedmanagementinformers
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/clientset/versioned/fake
+        alias: fakeseedmanagement
+      - pkg: github.com/gardener/gardener/pkg/client/settings/clientset/versioned
+        alias: settingsclientset
+      - pkg: github.com/gardener/gardener/pkg/client/settings/informers/externalversions
+        alias: settingsinformers
+      - pkg: github.com/gardener/gardener/pkg/client/settings/listers/settings/v1alpha1
+        alias: settingsv1alpha1lister
+      - pkg: github.com/gardener/gardener/pkg/controllerutils/(\w+)
+        alias: ${1}utils
+      - pkg: github.com/gardener/gardener/pkg/envtest
+        alias: gardenerenvtest
+      - pkg: github.com/gardener/gardener/pkg/extensions
+        alias: gardenerextensions
+      - pkg: github.com/gardener/gardener/pkg/gardenlet/bootstrap/util
+        alias: gardenletbootstraputil
+      - pkg: github.com/gardener/gardener/pkg/healthz
+        alias: gardenerhealthz
+      - pkg: github.com/gardener/gardener/pkg/mock/apiserver/authorization/authorizer
+        alias: mockauthorizer
+      - pkg: github.com/gardener/gardener/pkg/mock/\w+/(\w+)
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/core/v1
+        alias: mockcorev1
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/tools/(\w+)
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/util/(\w+)
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/operation/.*/(\w+)/mock$
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/operation/garden
```

Comment:
There is also `botanistpkg github.com/gardener/gardener/pkg/operation/botanist`.
Also does it make sense to combine this?

Reply:
Addressed it in 28bdc84e8ed0c83f370f0f7a6e9be2528fed85dc. PTAL.